### PR TITLE
extended vmss facts

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
Added additional fields that are important for users:
- tags
- id

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
azure_rm_virtualmachine_scaleset_facts
##### ANSIBLE VERSION
2.7

### CODE REVIEW

**Comment by zikalino** [comment]
currently vmss tests are disabled.
I have verified that new fields are returned correctly and facts module works (it passed ci tests)

File: lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py [hunk 1]
```diff
@@ -218,6 +224,10 @@
             type: str
             returned: always
             sample: Standard_D4
+        tags:
+            description: Tags assigned to the resource. Dictionary of string:string pairs.
+            type: dict
+            sample: { tag1: abc }
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py]: sample: { "tag1": "abc" }

**Review by yuwzho — APPROVED** [review]

**Comment by yuwzho** [comment]
shipit

File: lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py (hunk 1 above)
Comment (zikalino) [reply lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py]: oh, forgot again....

File: test/integration/targets/azure_rm_virtualmachine_scaleset/aliases [hunk 2]
```diff
@@ -2,3 +2,4 @@ cloud/azure
 shippable/azure/group4
 destructive
 disabled
+azure_rm_virtualmachine_scaleset_facts
```
Comment (yungezz) [inline test/integration/targets/azure_rm_virtualmachine_scaleset/aliases:5]: seems it's removed intended. are you sure want to add it back?

**Review by yungezz — APPROVED** [review]

**Pull request merged**